workitem.2.go

```diff
@@ -98,7 +99,12 @@ func (c *Workitem2Controller) List(ctx *app.ListWorkitem2Context) error {
 	if ctx.PageOffset == nil {
 		offset = 0
 	} else {
-		offset = int(*ctx.PageOffset)
+		offsetValue, err := strconv.Atoi(*ctx.PageOffset)
```

Comment:
It is. It's about negotiation;
Try https://api.github.com/search/repositories?q=a&page=safdsa&per_page=-2

I can't give you that, but here is what I can give you.